src/Http/Livewire/LivewireDatatable.php

```diff
@@ -1220,6 +1220,7 @@ public function buildDatabaseQuery($export = false)
     public function complexQuery($rules)
     {
         $this->complexQuery = $rules;
+        $this->setPage(1);
```

Comment:
Pagination is broken when using complex query because ```$this->setPage(1)``` is called on every refresh. Moved here so it is only called when the query is set